eq-author/src/App/introduction/Design/IntroductionEditor/CollapsiblesEditor/withCreateCollapsible.js

```diff
@@ -0,0 +1,39 @@
+import { graphql } from "react-apollo";
+import gql from "graphql-tag";
+import { filter } from "graphql-anywhere";
+
+const muatation = gql`
```

Comment:
Same as above, might be worth a find and replace for muatation seems to crop up in a couple places.